app/flatpak-builtins-utils.c

```diff
@@ -537,7 +537,7 @@ flatpak_resolve_matching_installed_refs (gboolean     disable_interaction,
           for (i = 0; i < ref_dir_pairs->len; i++)
             {
               RefDirPair *pair = g_ptr_array_index (ref_dir_pairs, i);
-              names[i] = flatpak_dir_get_name (pair->dir);
+              names[i] = g_strdup_printf (_("%s in %s"), pair->ref, flatpak_dir_get_name_cached (pair->dir));
```

Comment:
Perhaps we should put the dir name in parentheses for consistency with the single choice prompt above and the other `flatpak_resolve_*` prompts.

Reply:
Makes sense